### ISSUE TITLE
 mIoU on LIP validation

### ISSUE THREAD

**Issue by YaoooLiang** [PeikeLi/Self-Correction-Human-Parsing#39]
HI, I evaluate the pretrained weights [exp-schp-201908261155-lip.pth](https://drive.google.com/file/d/1k4dllHpu0bdx38J7H28rVVLpU-kOHmnH/view?usp=sharing) with the [evalute.py](https://github.com/PeikeLi/Self-Correction-Human-Parsing/blob/master/evaluate.py); but the miou is only 27.6% far away from 59.36%. The test parameters are the default parameters

**Comment by PeikeLi** [comment]
@YaoooLiang 

Please check your settings.
I just re-run the evaluation and list the details log as below.

```
python evaluate.py --model-restore ./pretrain_model/exp-schp-201908261155-lip.pth

image mean: [0.406, 0.456, 0.485]                                                                                                                                                                           
image std: [0.225, 0.224, 0.229]                                                                                                                                                                            
input space:BGR                                                                                                                                                                                             
BGR Transformation                                                                                                                                                                                          
Totoal testing sample numbers: 10000
                        
Pixel accuracy: 88.087233                                                                                                                                                                                   

Mean accuracy: 72.721079                                                                                                                                                                                    

Mean IU: 58.589516        
                                                                                                                                                                                  
OrderedDict([('Background', 88.3442361878685), ('Hat', 69.96947809134201), ('Hair', 73.54309620912845), ('Glove', 50.30418305554127), ('Sunglasses', 40.64712540171674), ('Upper-clothes', 69.88614578224038│·························
), ('Dress', 39.12394794248705), ('Coat', 57.426223799119505), ('Socks', 54.19401400183561), ('Pants', 75.97924333900869), ('Jumpsuits', 32.68272162400498), ('Scarf', 26.34789636820655), ('Skirt', 31.5923│·························
39627695193), ('Face', 76.19456391929387), ('Left-arm', 68.6327781746178), ('Right-arm', 70.9168761750908), ('Left-leg', 67.22625658157922), ('Right-leg', 66.52695495012838), ('Left-shoe', 55.736763623158│·························
82), ('Right-shoe', 56.51546955975324), ('Pixel accuracy', 88.08723287463441), ('Mean accuracy', 72.72107948118848), ('Mean IU', 58.58951572069085)]) 
```

**Comment by YaoooLiang** [comment]
I have used torch.nn.BatchNorm2d instead of InPlaceABNSync, that's may be the reason. Thank you for quickly reply

**Comment by PeikeLi** [comment]
Actually, InPlaceABNSync = torch.nn.BatchNorm2d + torch.nn.LeakyRelu

**Comment by YaoooLiang** [comment]
Thank you, I will try this
